medusa/logger/__init__.py

```diff
@@ -68,7 +70,15 @@
 def rebuild_censored_list():
     """Rebuild the censored list."""
     # set of censored items
-    results = {value for value in itervalues(censored_items) if value}
+    results = set()
+    for value in itervalues(censored_items):
+        if not value:
+            continue
+        if isinstance(value, collections.Iterable) and not isinstance(value, six.string_types):
+            results.update(value)
+        else:
+            results.add(value)
+
```

Comment:
@labrys this is what it came down to.
Also I noticed that the rebuild_censored_list() function was called for each check_setting_str() and check_setting_list() function. I reduced that to 1 call. Should improve startup times.